spec/javascript/components/ShowLocationTile.js

```diff
@@ -0,0 +1,33 @@
+import ShowLocationTile from '../../../app/javascript/react/components/ShowLocationTile'
+import React from 'react';
+
+describe('ShowLocationTile', () => {
+  let name,
+      address,
+      city,
+      state,
+      zip,
+      wrapper
+
+      beforeEach(() => {
+        wrapper = mount (
+          <ShowLocationTile
+          name="Trillium"
+          address="That big street in Boston"
+          city="Boston"
+          state="MA"
+          zip="02420"
+          />
+        );
+    })
+
+  it("Should have an h1 tag", ()=>{
```

Comment:
This isn't sufficiently tested -- you should be testing that the appropriate text displays, not just that there are h1 and p tags